sim/core/aura.go

```diff
@@ -1003,9 +1003,11 @@ func ApplyFixedUptimeAura(aura *Aura, uptime float64, tickLength time.Duration)
 				if sim.RandomFloat("FixedAura") < uptime {
 					// Use random duration to compensate for increased chance collapsed into single tick.
 					randomDur := tickLength + time.Duration(float64(auraDuration-tickLength)*sim.RandomFloat("FixedAuraDur"))
+					oldDur := aura.Duration
```

Comment:
Original duration is already saved in a local variable on line 984. Not sure if it might be more efficient to capture 1 less external variable though. W/e.

Reply:
4b65f50e86602d9d83103bf815d75845ceaa7329